app/views/comment_mailer/edited_comment_notification.text.erb

```diff
@@ -8,4 +8,5 @@
 
 <%= to_plain_text(raw(@comment.sanitized_mailer_content)) %>
 
-<%= render 'comment_notification_footer', formats: [:text] %><% end %>
+<%= render "comment_notification_footer", formats: [:text] %>
+<% end %>
```

Comment:
I think this was like this because of whitespace? For these text emails any whitespace in here ends up in the email itself. Might need to end the render tag with `-%>` to make sure there is no extra newline after it

(Quick check with the mailer previews shows that there is indeed an extra newline here)